docs-2.0/3.ngql-guide/8.clauses-and-options/return.md

```diff
@@ -0,0 +1,233 @@
+# RETURN
+
+The `RETURN` clause defines the output of openCypher statements such as `MATCH` or `UNWIND`. To return multiple fields, separate them with commas.
+
+## OpenCypher compatibility
+
+This topic applies to the openCypher syntax in nGQL only. For nGQL extensions, use [`YIELD`](yield.md).
```

Comment:
This topic applies to the openCypher syntax in Nebula Graph. WDYT

Reply:
At first sight, I thought it is a good idea. But on second thought, this is the nGQL guide and the openCypher part still belongs to nGQL 2.0... So why not nGQL

nGQL 2.0 = nGQL extensions + openCypher, am I right?